recipes/codelldb/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+
+if [[ ${target_platform} == "osx-arm64" ]]; then
+    export CMAKE_ARGS="-DLLVM_TRIPLE='aarch64-apple-darwin' ${CMAKE_ARGS}"
+fi
+
+cmake -S . -B build \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_VERBOSE_MAKEFILE=ON \
+    -Wno-dev \
+    -DBUILD_TESTING=OFF \
+    -DLLDB_PACKAGE=${BUILD_PREFIX} \
+    ${CMAKE_ARGS}
+
+cmake --build build -j${CPU_COUNT}
+
```

Comment:
Any `ctest` or make `check`?

Reply:
While the `CMakeLists.txt` for this package does have a couple of test targets, all of them appear to require quite a complex setup that would be very difficult to set up in CI.

Reply:
Can you elaborate? How complex?